### DISCUSSION TITLE
Service logs error in runtime area

### DISCUSSION THREAD

**Discussion by fabioS24** [mia-platform/community#257] [Q&A] [answered]
### Description

Hi!

When I try to open the logs from the runtime area by selecting multiple pods it says "Refresh me". But if I try to open the logs from the pod they show fine. Logs for multiple pods seem to not work correctly. 

The same behavior occurs if you select only one pod from the list.

**Comment by fredmaggiowski** [comment, marked as answer]
It seems there is a bug in the multi-container logs @jacoposte @albertotessarotto can you track it so that we can solve it asap?

**Reply by jacoposte** [reply]
Yes, we tracked it and we will solve it asap
